misc/python/materialize/test_analytics/connector/test_analytics_connector.py

```diff
@@ -216,7 +217,7 @@ def _execute_sql(
 
         try:
             start_time = time.time()
-            cursor.execute(sql)
+            cursor.execute(sql.encode("utf-8"))
```

Comment:
I wrote `encode("utf-8")` everywhere and the linter did that. My best guess is that in some cases Python can infer that this file will always be run with utf-8 default encoding, but for library-like code in misc it can't.

Reply:
Interesting!

Reply:
Doesn't `encode()` always default to `utf-8`, regardless of where it's called?

Reply:
Since Python3 yes, in Python2 it still was dependent on the setting:

> Default encoding is the current default string encoding

I'm not sure if that's the explanation for why the linter keeps the "utf-8" values in some cases though. I'll just remove all the "utf-8" explicit mentions.